src/pages/endpoint/Endpoint.scss

```diff
@@ -0,0 +1,60 @@
+#Endpoint {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+  }
+
+  .close-button {
+    position: absolute;
+    right: 4px;
+    top: 120px;
+    cursor: pointer;
+  }
+
+  .flag {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    margin-top: 72px;
+    margin-bottom: 24px;
+  }
+
+  .detail-tile-row {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+  }
+
+  .detail-tile {
+    height: 51px;
+    display: flex;
+    flex-direction: row;
+
+    span {
+      margin-right: 0px;
+      font-size: 14px;
+      font-weight: 400;
+    }
+
+    label {
+      left: 6px;
+      margin-top: -10px;
+    }
+
+    svg {
+      margin-left: 16px;
+    }
+  }
+
+  .title-tile {

```

Comment:
Can we make sure that these labels are centered => 
<img width="771" alt="Screen Shot 2021-05-04 at 8 41 05 AM" src="https://user-images.githubusercontent.com/13072035/117021466-9cd18680-acb4-11eb-8c52-a1d7ebbd7b17.png">

Reply:
In the design, the titles were aligned to the top and left, and the information to the right. Then should we make them centered on wider screens and like this on mobile screens?